Source/WebCore/platform/gamepad/mac/HIDGamepadProvider.mm

```diff
@@ -270,8 +270,12 @@ static GameControllerFrameworkHandlesDevice gameControllerFrameworkWillHandleHID
 
     if (!removedGamepad) {
 #if HAVE(MULTIGAMEPADPROVIDER_SUPPORT)
+#if ASSERT_ENABLED
```

Comment:
I don't think avoiding a RetainPtr is worth the extra code duplication here.

Reply:
Fair enough. Though what is even the point of assigning to a local variable here?